app/Insightly/Resources/ProjectResource.php

```diff
@@ -7,16 +7,19 @@
 use App\Domain\Contacts\ContactType;
 use App\Domain\Integrations\Integration;
 use App\Insightly\Objects\ProjectStage;
+use App\Insightly\Objects\ProjectState;
 
 interface ProjectResource
 {
     public function create(Integration $integration): int;
 
-    public function updateWithCoupon(int $insightlyId, string $couponCode): void;
+    public function updateWithCoupon(int $projectId, string $couponCode): void;
```

Comment:
I noticed we now have 4 flavors inside the resource interfaces for the Insightly id:
- `id`
- `insightlyId`
- `projectId`

I would pick one and make it consistent over all the interfaces.

Reply:
How will we handle `ContactResource` and `OrganizationResource`?

I think that using `id` is actually just fine. We are in the context of Insightly and all the other interfaces like the repositories use `id` for the object that is manipulated. For example `ContactRepository::getById(UuidInterface $id)`